NTupleAnalyzer_nanoAOD.py

```diff
@@ -1673,6 +1878,11 @@ def FullKinematicCalculation(T,variation):
 	_DPhiu2j1 = abs(muons[1].DeltaPhi(jets[0]))
 	_DPhiu2j2 = abs(muons[1].DeltaPhi(jets[1]))
 
+	year = '2016' #Fixme need to integrate year option
```

Comment:
this is fine for now, but we need to make this a global option eventually

Reply:
I suppose the samples will be sorted into directories by year at some stage--could we just pull the year from the path? Or, would it be better to specify as a command line option?

Reply:
We can decide later, not a problem. Not sure yet which is best...